library/common/main_interface.h

```diff
@@ -30,6 +30,7 @@ envoy_stream_t init_stream(envoy_engine_t);
  * can occur.
  * @param stream, handle to the stream to be started.
  * @param callbacks, the callbacks that will run the stream callbacks.
+ * @param options, DEPRECATED.
```

Comment:
https://github.com/lyft/envoy-mobile/issues/643

cc @buildbreaker @rebello95

Reply:
sounds good, not opposed to removing as part of this PR either